operations/research-and-development/product/release-process/cloud-release.md

```diff
@@ -0,0 +1,33 @@
+# Cloud Release Process
+
+This document outlines the Cloud release process for the Mattermost core team. Our release process draws from many best practice Agile, Scrum, and Software Development Lifecycle methodologies.
+
+## Release Timeline
+
+Notes:
+- All cut-off dates are based on 10am in San Francisco [(Pacific Time](http://everytimezone.com/)) on the day stated.
+- T-minus counts are measured in "working days" (weekdays other than major holidays concurrent in US and Canada) prior to release day.
+
+### A. (T-minus 8 working days) Fast-forward master to cloud branch
```

Comment:
I know we've been calling it fast-forward but technically it's not and is just a merge 

```suggestion
### A. (T-minus 8 working days) Merge master to cloud branch
```

Reply:
Wouldn't the new schedule of cut/merge on Monday and release on Wednesday make this t-minus 9 days?

Reply:
My bad - updated.